conf/base.config

```diff
@@ -95,9 +95,15 @@ process {
     errorStrategy = { task.exitStatus in [1] ? 'retry' : 'finish' } 
   }
 
+  // other process specific exit statuses
+  withName: nuclear_contamination {
```

Comment:
But this doesn't do anything with dedup?

Reply:
lel. That was in another already merged PR. This just removes the validExtStatus stuff